setup.py

```diff
@@ -42,6 +42,7 @@
     'PyYAML>=3.11,<3.13',
     'lxml>=3.8,<3.9',
     'draftjs_exporter[lxml]>=1.1.0,<2.0',
+    'python-twitter>=3.3',
```

Comment:
would be better to fix the version to `<4.0` or something, so it won't break when they release an incompatible version